h2o-py-cloud-extensions/README.md

```diff
@@ -26,19 +26,19 @@ h2o.init()
 
 Set a H2O Cloud instance you will be connecting to:
 ```
-hce.settings.connection.client_id = 'q8s-qa-platform'
-hce.settings.connection.token_endpoint_url = 'https://auth.demo.h2o.ai/auth/realms/q8s-qa/protocol/openid-connect/token'
+hce.settings.connection.client_id = 'hac-platform-public'
+hce.settings.connection.token_endpoint_url = 'https://auth.internal.dedicated.h2o.ai/auth/realms/hac/protocol/openid-connect/token'
 ```
 
-Get authenticated against H2O.ai cloud (https://cloud-qa.h2o.ai/auth/get-platform-token) and set platform token:
+Get authenticated against H2O.ai cloud (https://internal.dedicated.h2o.ai/auth/get-platform-token) and set platform token:
 ```
 hce.settings.connection.refresh_token = "TOKEN_THAT_YOU_RECIEVED_AFTER_AUTHENTICATION"
 ```
 
 ### MLOps
 Set MLOps instance and the project that you will utilize for publishing your models.
 ```
-hce.settings.mlops.api_url = 'https://mlops-api.cloud-qa.h2o.ai'
+hce.settings.mlops.api_url = 'https://mlops-api.internal.dedicated.h2o.ai'
```

Comment:
Would be great to just change at one place when the web address strings change in the future.

Reply:
I hope this won't change so often. I would like to keep  jupyter notebooks easy to use and be directly downloadable from github for now. Once we get to the final state ,we can generate doc  and make example notebooks downloadable from h2o documentation website. In such a stage, the addresses could be defined in some `build.gradle` file.